db-lib/db/private/postgresql/dbsystem.rkt

```diff
@@ -228,6 +234,9 @@
   (2278 void              #f  1 recv-void   #f)
   (2287 record-array      #f  1 recv-array  #f)
 
+  
+  
```

Comment:
you've got a couple of spots where you're adding seemingly unnecessary blank lines. i mention this only because i am bored at work.